src/qt/test/test_main.cpp

```diff
@@ -70,7 +70,7 @@ int main(int argc, char* argv[])
     BitcoinApplication app(*node);
     app.setApplicationName("Bitcoin-Qt-test");
 
-    node->setupServerArgs();            // Make gArgs available in the NodeContext
+    node->context()->args = &gArgs;     // Make gArgs available in the NodeContext
     node->context()->args->ClearArgs(); // Clear added args again
```

Comment:
This line won't be needed anymore

Reply:
re: https://github.com/bitcoin-core/gui/pull/35#discussion_r456258986

> This line won't be needed anymore

Thanks, removed